source/agora/flash/Channel.d

```diff
@@ -1157,13 +1185,24 @@ LOuter: while (1)
     {
         const old_balance = this.cur_balance;
 
+        const old_seq_id = this.cur_seq_id;
```

Comment:
`collectSignatures` already takes `update.seq_id` as an argument. Could we wait until `collectSignatures` to return to update the `cur_seq_id`?

Reply:
Sure, good idea.